### ISSUE TITLE
Standards and conditions fields should match in builder and export views

### ISSUE THREAD

**Issue by beparticular** [CMSgov/eAPD#2519]
In our debrief with ID reviewers on their re-pilot submission, there was some confusion around the standards and conditions fields and answers that were automatically generated in the export view. 

Considerations 
The fields in the builder view don't exactly have labels to match up anything to the export view. 
Maybe if the description is filled out for how the activity supports the s/c, the question/field for this does not meet s/c should not be shown on the export view? 
Should these fields have proper labels, or does the current question suffice? 

Builder 
![Screen Shot 2020-09-16 at 6.02.21 PM.png](https://images.zenhubusercontent.com/591f3b9351aa2c30577ecf92/15b4c2ad-785e-4bf5-9780-63a0d8c0f081)

Export 
![Screen Shot 2020-09-16 at 6.03.12 PM.png](https://images.zenhubusercontent.com/591f3b9351aa2c30577ecf92/58f6da34-0673-4007-965b-e9b9e92bc45c)

### This task is done when...
 - [ ] Designs are created for s/c that match in the builder and export and are not confusing to reviewers or states. 
 - [ ] Designs are approved by product

**Comment by jjames521** [comment]
@beparticular 

I considered the fact that two options are currently being displayed in the builder at the same time, with variances in text box fields.  One is TinyMCE and the other is a regular field usually used for short radio button responses.  

This lead me to think that we could trim down the confusion by taking down both the inconsistencies of styling and the extra copy.  

Providing the user with radio buttons to the Yes/No question with an explanation on why/why not that reflects in the export view and changes based on the users response.

The radio button would default on why this activity does support 42 CFR 433.112 the new interaction provides a clear direction.  

**Builder View**
![Standards & Conditions Builder View.png](https://images.zenhubusercontent.com/5e7151677c78c820a2a11970/f8524651-fc5f-4c73-b195-afda76886152)

**Export View**
![Standards & Conditions Export View.png](https://images.zenhubusercontent.com/5e7151677c78c820a2a11970/67864944-c6a7-4bae-b322-5a0a13c593a4)

**Comment by beparticular** [comment]
I like this approach - 
@CSwartzHMA @NAretakis do you see any reason we wouldn't be able to move in this direction?

**Comment by CSwartzHMA** [comment]
@jjames521 So, if yes, the export would indicate this activity does not support and then display whatever was entered in the text box?

**Comment by jjames521** [comment]
@CSwartzHMA Yep!

**Comment by CSwartzHMA** [comment]
@beparticular I believe there was a conversation at one point that we DID want an explanation as to how the activity supports Medicaid S&Cs, so I am worried that the "please explain why" would be interpreted as only needed if the answer is no and would be left blank. I think that was the reason for the redundant questions. I like the mechanics, but maybe we could just reword this to be clear that a description is required regardless of the radio button selected.

**Comment by CSwartzHMA** [comment]
Perhaps @NAretakis has more insight into how important these narratives are. I think we hear during research that the answers are copied and pasted year over year anyway.

**Comment by jjames521** [comment]
@CSwartzHMA @beparticular 

Here are some variant stylings, one of the option is having the phrasing change from "Please explain why" to "Please Explain Why Not"

![Standards & Conditions Builder View01.png](https://images.zenhubusercontent.com/5e7151677c78c820a2a11970/fc791ea6-f293-4bdf-836a-dddce1e979fe)

![Standards & Conditions Builder View02.png](https://images.zenhubusercontent.com/5e7151677c78c820a2a11970/c1ab11b8-eb9c-4eea-a97d-fd448c202eaf)

![Standards & Conditions Builder View03.png](https://images.zenhubusercontent.com/5e7151677c78c820a2a11970/65fb6bd9-3ff7-4fa2-bcbc-65a684bc2ee0)

![Standards & Conditions Builder View04.png](https://images.zenhubusercontent.com/5e7151677c78c820a2a11970/296e0b40-1f20-400d-873b-905592fc32f9)

![Standards & Conditions Builder View05.png](https://images.zenhubusercontent.com/5e7151677c78c820a2a11970/e61e7dbb-8763-46dd-94ec-24db3cbdbe48)

**Comment by beparticular** [comment]
I think we can address the validations when we address validations as a whole since we don't have any mechanism for required fields at this time. 
Right now it's not a field that is always being filled out anyways, and I think that might be because it's confusing as designed now.

After discussion with Jesse, let's show the field without Yes or No selected, then once the selection is made show the editor field with Please explain why or Please explain why not.

**Comment by CSwartzHMA** [comment]
Perfect. I like that approach.

**Comment by beparticular** [comment]
@thetif this can be moved forward to dev!
